py/nativeglue.h

```diff
@@ -145,7 +145,7 @@ typedef struct _mp_fun_table_t {
     #if defined(__GNUC__)
     NORETURN // Only certain compilers support no-return attributes in function pointer declarations
     #endif
-    void (*raise_msg_str)(const mp_obj_type_t *exc_type, const char *msg);
```

Comment:
Does raising exceptions from a native module actually work with these changes? I expect that it doesn't, because there's no way for the string to be entered in the list of translations that is baked into the core, and the `translate()` function is not available to call from the dynamic runtime either.